Creative_backery_layout_v-1

ZupaFly opened the pull request:
[DEMO LINK](https://ZupaFly.github.io/layout_creativeBakery/)

Volodymyr-Melnyk-Fron10der reviewed (changes requested):
Well done💪🏼
1. content should be centered in button
![image](https://user-images.githubusercontent.com/122989585/221349356-7a09269c-d5e0-47e1-9795-0cbbece49ab0.png)

ZupaFly commented:
> 

Thank you for your comment. I see it now. Changed it. Now looks similar to design.

polosanya reviewed (changes requested):
Looks great! Found a few comments that would be great to fix
1. Let's add hover effects to these links
![image](https://user-images.githubusercontent.com/96409563/221403072-78d0855f-43af-434b-b649-cfe9d58b3f7c.png)
2. This title should be black if you are implementing a black theme
![image](https://user-images.githubusercontent.com/96409563/221403142-c23239f5-8fb4-4701-999a-bf3d287201e4.png)
![image](https://user-images.githubusercontent.com/96409563/221403159-f5fa2089-78c5-42e1-9305-96d613b05c6c.png)
3. Title doesn't match the design
![image](https://user-images.githubusercontent.com/96409563/221403192-2299b8b0-b5fa-49a8-9b07-c30df821ebc6.png)
![image](https://user-images.githubusercontent.com/96409563/221403196-bfc4a6f9-8cd5-40b1-98e9-934c462d5e3f.png)
You can also add some hover effects to statistic numbers
4. Let's centre these links vertically
![image](https://user-images.githubusercontent.com/96409563/221403281-9b00369f-ee3c-4665-83c2-3a8125f56506.png)
![image](https://user-images.githubusercontent.com/96409563/221403288-28d955c9-6ef5-4d4b-9cec-9af6c176b4c6.png)
5. Let's make burger menu icon black
![image](https://user-images.githubusercontent.com/96409563/221403299-f7e0b122-d666-4c1a-9c86-a98db57c3f31.png)
6. Try to centre the cards
![image](https://user-images.githubusercontent.com/96409563/221403405-59def00f-5be1-4793-a07f-095a1b328baa.png)
![image](https://user-images.githubusercontent.com/96409563/221403414-95ca5e1b-1f52-49ff-a441-0d19e8e7b435.png)

ZupaFly commented:
> Looks great! Found a few comments that would be great to fix
> 
> 1. Let's add hover effects to these links
>    ![image](https://user-images.githubusercontent.com/96409563/221403072-78d0855f-43af-434b-b649-cfe9d58b3f7c.png)
> 2. This title should be black if you are implementing a black theme
>    ![image](https://user-images.githubusercontent.com/96409563/221403142-c23239f5-8fb4-4701-999a-bf3d287201e4.png)
>    ![image](https://user-images.githubusercontent.com/96409563/221403159-f5fa2089-78c5-42e1-9305-96d613b05c6c.png)
> 3. Title doesn't match the design
>    ![image](https://user-images.githubusercontent.com/96409563/221403192-2299b8b0-b5fa-49a8-9b07-c30df821ebc6.png)
>    ![image](https://user-images.githubusercontent.com/96409563/221403196-bfc4a6f9-8cd5-40b1-98e9-934c462d5e3f.png)
>    You can also add some hover effects to statistic numbers
> 4. Let's centre these links vertically
>    ![image](https://user-images.githubusercontent.com/96409563/221403281-9b00369f-ee3c-4665-83c2-3a8125f56506.png)
>    ![image](https://user-images.githubusercontent.com/96409563/221403288-28d955c9-6ef5-4d4b-9cec-9af6c176b4c6.png)
> 5. Let's make burger menu icon black
>    ![image](https://user-images.githubusercontent.com/96409563/221403299-f7e0b122-d666-4c1a-9c86-a98db57c3f31.png)
> 6. Try to centre the cards
>    ![image](https://user-images.githubusercontent.com/96409563/221403405-59def00f-5be1-4793-a07f-095a1b328baa.png)
>    ![image](https://user-images.githubusercontent.com/96409563/221403414-95ca5e1b-1f52-49ff-a441-0d19e8e7b435.png)

Thanks for your comments:
1. Added underline link on hover.
2. Oh, my bad. Maybe i just checked pink design that time. Changed to black color.
3. I am not sure i got that comment right. To position that block i used display: grid 
![image](https://user-images.githubusercontent.com/106464335/221416227-ac9aca80-32f0-42a7-b026-1102ffeead6f.png)
![image](https://user-images.githubusercontent.com/106464335/221416534-575c1ad5-8018-4f21-b96d-c2c5b47db3ff.png)

And that grid display perfectly matches design on 1440px display. If i will stretch it more than 1440 than my title block also became bigger. Does that mean that i should hardcode width of that block with max-width? 
Added hover effect to static numbers.
4. Done
5. Its black now...feel messy working with 2 identical designs...
6. And fixed the centering of product blocks.

Thanks, and waiting for your comments.

polosanya reviewed (approved):
Well done! You did a great job! 🔥 
3. Yes, you can try to limit the width, so this title looks the same on large screens too, but it's not critical. It's up to you)
![image](https://user-images.githubusercontent.com/96409563/221502257-78c520b3-1873-4a9e-a23d-084925aba403.png)

ZupaFly commented:
> Well done! You did a great job! 🔥 3. Yes, you can try to limit the width, so this title looks the same on large screens too, but it's not critical. It's up to you) ![image](https://user-images.githubusercontent.com/96409563/221502257-78c520b3-1873-4a9e-a23d-084925aba403.png)

Thanks for comments and approval. I guess it's up to the customer, but i've got the point :)
